zap/src/test/java/org/zaproxy/zap/utils/ByteBuilderUnitTest.java

```diff
@@ -65,4 +69,314 @@ public void shouldBeInitializedWithDoubleCapacityOfGivenArray() {
         // then
         assertThat(capacity, is(2 * TEST_ARRAY.length));
     }
+
+    @Test
+    public void shouldIncreaseCapacityFrom5To20() {
```

Comment:
It should be e.g. `shouldIncreaseCapacityWhenBiggerThanActual`, which values are used it's not important. (Ideally it should have other tests with negative and lower than actual capacity.)

Reply:
Test name changed